docs/index.md

```diff
@@ -17,14 +17,16 @@ terraform {
   required_version = ">= 0.13.0"
   required_providers {
     edgecenter = {
-      source  = "Edge-Center/edgecenter"
-      version = "0.3.18"
+      source = "local.edgecenter.ru/repo/edgecenter"
```

Comment:
What is the source address? Is this a path to a folder, or an url?

Reply:
Terraform registry or path to folder in .terraform.d/plugins